tests/integration/sortable-group-test.js

```diff
@@ -0,0 +1,135 @@
+import hbs from 'htmlbars-inline-precompile';
+import { test, moduleForComponent } from 'ember-qunit';
+import Ember from 'ember';
+
+const { A, run } = Ember;
+const a = A;
+
+function itemLabels() {
+  return $('.sortable-item .label').toArray().map(function(item) {
+    return $(item).text();
+  });
+}
+
+function itemHeight() {
+  return $('.sortable-item').outerHeight();
+}
+
+moduleForComponent('sortable-group', 'SortableGroup', { integration: true });
+
+test('Drag an item', function(assert) {
+  $('#ember-testing').css('zoom', '100%');
```

Comment:
This is a bit of a hack... Without this, the items get adjusted to weird positions while dragging. I assume somewhere there is some positioning code that doesn't account for the zoom.